src/Microsoft.Health.Dicom.Functions/Update/UpdateOptions.cs

```diff
@@ -0,0 +1,41 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.Health.Operations.Functions.DurableTask;
+
+namespace Microsoft.Health.Dicom.Functions.Update;
+
+/// <summary>
+/// Represents the options for a "update" function.
+/// </summary>
+public class UpdateOptions
+{
+    internal const string SectionName = "Update";
+
+    /// <summary>
+    /// Gets or sets the number of DICOM instances processed by a single activity.
```

Comment:
This comment is necessarily true -- these batches are within a single activity. So perhaps we should change the description and names to avoid ambiguity with the other functions that use these names

Reply:
Updated comment